### DISCUSSION TITLE
Embedding project view

### DISCUSSION THREAD

**Discussion by iurimatias** [marcusolsson/obsidian-projects#660] [Help]
Is there currently a way to embed a project and also a view in a obsidian note? This would be quite useful for dashboards and I would also like to see multiple projects in the same note.

e.g something like `![[MyProject#GalleryView]]`

I tried using Obsidians [embed syntax](https://help.obsidian.md/Linking+notes+and+files/Embedding+files#Embed+a+note+in+another+note) which works in other plugins ([example](https://dataloom.xyz/other/embedding-looms/)) without much luck but wondering if there is a way to do this I'm missing.

**Comment by Acylation** [comment]
#39 shows an early implementation integration in codeblock, you can track the request there for current progress. It won't be worked on in the coming few weeks unfortunately. The feature requires us adapting to smaller ranges, and may be helpful in the mobile versions, panels and canvas integration.
